app/models/user/permissions.rb

```diff
@@ -89,22 +88,34 @@ def services_member_permission
   end
 
   def has_access_to_service?(service)
-    services_permission = services_member_permission
-    services_permission && services_permission.has_service?(service) || has_access_to_all_services?
+    services_member_permission&.has_service?(service) || has_access_to_all_services?
```

Comment:
I see you have changed the implementation of `has_access_to_all_services?` to make sense. Perhaps change order also because reading like this I was confused the logic maybe not being the intended one.

```suggestion
    has_access_to_all_services? || services_member_permission&.has_service?(service)
```